scripts/oidc-conformance-run.sh

```
@@ -9,7 +9,10 @@ SERVER="$REPO/target/release/preloop-server"
 RUNNER="$REPO/target/release/preloop-runner"
 PORT=9192
 BASE="http://127.0.0.1:$PORT"
-SYSTEM_TOKEN="${PRELOOP_SYSTEM_TOKEN:-preloop-system-token}"
+if [[ -z "${PRELOOP_SYSTEM_TOKEN:-}" ]]; then
+    export PRELOOP_SYSTEM_TOKEN="$(python3 -c 'import secrets; print(secrets.token_hex(32))')"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)